services/agents/nodes/classes/text/build-template.ts

```diff
@@ -9,7 +9,7 @@ import {
 	requestPortMessages,
 	requests,
 } from "@/drizzle";
-import { logger } from "@trigger.dev/sdk/v3";
+// import { logger } from "@trigger.dev/sdk/v3";
```

Comment:
nits: I thought it might be fine to delete all the commented-out lines in this file.